src/components/v5/shared/Button/PendingButton.tsx

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable react/button-has-type */
+import React, { FC, PropsWithChildren } from 'react';
+import { useIntl } from 'react-intl';
+import clsx from 'clsx';
+
+import { PendingButtonProps } from './types';
+import SpinnerLoader from '~shared/Preloaders/SpinnerLoader';
+import styles from './PendingButton.module.css';
+import Icon from '~shared/Icon';
+
+const displayName = 'v5.Button.PendingButton';
```

Comment:
v5.shared.Button.PendingButton

Reply:
As I can see, we have never used the prefix `shared` for components inside this directory.